packages/video_player/video_player_platform_interface/pubspec.yaml

```diff
@@ -3,16 +3,16 @@ description: A common platform interface for the video_player plugin.
 homepage: https://github.com/flutter/plugins/tree/master/packages/video_player/video_player_platform_interface
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 3.0.0-nullsafety.3
+version: 4.0.0-nullsafety.0
 
 dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0-nullsafety.3
-
-dev_dependencies:
```

Comment:
`flutter_test` is now a `dependency`?

Reply:
yeah, I dunno how else to do it. The point of the pigeon change was that I'm moving the test APIs from package:flutter to package:flutter_test, but the side-effect of that is that that means that the test APIs that are generated by pigeon for plugins don't work unless they import flutter_test. I mean, this was always broken really (in that we always had non-dev code depending on test framework code, it's just that the test framework code wasn't in flutter_test). I don't really know how else to update the packages. I suppose we could have a test package with the test API and a non-test package with the prod API but that seems excessive.

(also posted to #hackers-ecosystem)

Reply:
got it. Thanks for the explanation. It would be solved if Dart had remote file imports like golang or [Deno](https://deno.land/manual/examples/import_export#remote-import), so there's no need for dev vs non-dev dependencies, just an entrypoint and run.